presto-parser/src/main/java/com/facebook/presto/sql/tree/AtTimeZone.java

```diff
@@ -41,7 +41,7 @@ public AtTimeZone(NodeLocation location, Expression value, Expression timeZone)
     private AtTimeZone(Optional<NodeLocation> location, Expression value, Expression timeZone)
     {
         super(location);
-        checkArgument(timeZone instanceof IntervalLiteral || timeZone instanceof StringLiteral, "timeZone must be IntervalLiteral or StringLiteral");
+        checkArgument(timeZone instanceof IntervalLiteral || timeZone instanceof StringLiteral || timeZone instanceof Parameter, "timeZone must be IntervalLiteral, StringLiteral, or Parameter");
```

Comment:
Why do we need this change?

Reply:
Since we replace String and Interval literals with Parameter in a sql template, we need to allow the timeZone to be an instance of Parameter.

Reply:
I see, so let's add tests for AtTimeZone node like the other literals

Reply:
Added a test for AtTimeZone expression.